lavamoat/browserify/policy-override.json

```diff
@@ -68,6 +68,35 @@
       "globals": {
         "crypto": true
       }
+    },
+    "@rive-app/react-canvas>@rive-app/canvas": {
+      "globals": {
+        "AudioContext": false,
+        "DOMMatrix": false,
+        "FinalizationRegistry": false,
+        "Iterator": false,
+        "MouseEvent": false,
+        "ResizeObserver": false,
+        "crypto": false,
+        "define": false,
+        "devicePixelRatio": false,
+        "h": false,
+        "importScripts": false,
+        "location.href": false,
+        "prompt": false,
+        "readline": false,
+        "webkitAudioContext": false
```

Comment:
(blocker) network access and `document` are missing from the falsy overrides.

We agreed we need to forbid network access and limit which `document.*` methods are available. 

Is there a reason to back out of this? What other guarantees do we have that the package is not capable of fetching content and executing it without these?